### PR TITLE
Tidy up QuorumTransactionManager to support chainId and madatoryFor

### PR DESCRIPTION
**Pull request by antonydenyer**
### What does this PR do?

- Enable the specification of ChainId in transactions when using QuorumTransactionManager
- Enable mandatoryFor in the QuorumTransactionManager

### CODE REVIEW

**Comment by antonydenyer** [comment]
@AlexandrouR any chance you could take a look?

**Comment by AlexandrouR** [comment]
> 

Sure I will also ask @andrii-kl  to have a look.

File: src/main/java/org/web3j/quorum/methods/request/PrivateRawTransaction.java [hunk 1]
```diff
@@ -55,11 +55,15 @@ public void setPrivateFor(List<String> privateFor) {
...
+    public int getPrivacyFlag() {
+        if (privacyFlag != null) {
+            return privacyFlag.getValue();
+        }
+
+        return PrivacyFlag.STANDARD_PRIVATE.getValue();
     }
 
-    public void setPrivacyFlag(PrivacyFlag privacyFlag) {
+    public void setPrivacyFlag(PrivacyFlag PrivacyFlag) {
```
Comment (AlexandrouR) [inline src/main/java/org/web3j/quorum/methods/request/PrivateRawTransaction.java:66]: Should this be Capital?

File: src/main/java/org/web3j/quorum/methods/response/ContractPrivacyMetadataInfo.java [hunk 2]
```diff
@@ -77,7 +77,7 @@ public PrivacyFlag getPrivacyFlag() {
             return privacyFlag;
         }
 
-        public void setPrivacyFlag(PrivacyFlag privacyFlag) {
+        public void setPrivacyFlag(PrivacyFlag PrivacyFlag) {
```
Comment (AlexandrouR) [inline src/main/java/org/web3j/quorum/methods/response/ContractPrivacyMetadataInfo.java:80]: Same as above

File: src/main/java/org/web3j/quorum/methods/request/PrivateRawTransaction.java (hunk 1 above)
Comment (antonydenyer) [reply src/main/java/org/web3j/quorum/methods/request/PrivateRawTransaction.java]: :facepalm: fat fingers, fixed.

**Comment by AlexandrouR** [comment]
Looks great tests are passing i will make a PR later to remove Travis and add GitHub actions

**Comment by antonydenyer** [comment]
@AlexandrouR any ideas when this will be released?

**Comment by AlexandrouR** [comment]
@antonydenyer a snapshot version was released for 4.8.9-SNAPSHOT but some of the integration tests are failing. I am looking at how I can automate the IT in GitHub actions using quorum.

**Pull request merged**